js/utils.js

```diff
@@ -974,6 +974,135 @@ Fliplet.Registry.set('dynamicListUtils', (function () {
     return Promise.resolve(records);
   }
 
+  /**
+   * Sort items by field witch user have selected
+   * 
+   * @param {Object} options - object with settings for sort list items
+   *        keys: 
+   *          sortField {String} - name of the field to sort
+   *          sortOrder {String} - sort order of
+   *          records {Array} - array of records to sort
+   * @return {Array} - sorted by field array
+   */
+  function sortByField(options) {
+    // If user doesn't set sorting do nothing
+    if (!options.sortField) {
+      return options.records;
+    }
+  
+    var records = JSON.parse(JSON.stringify(options.records));
```

Comment:
Create an array of 10 million entries, each with 10 properties.

Test which one is faster:

- `JSON.parse(JSON.stringify(arr))`
- `_.clone(arr)`

Reply:
`_.clone` will be better in that situation, `JSON.parse(JSON.stringify(arr))` I'll change it to clone.